docs/source/app-auth-update.rst

```diff
@@ -0,0 +1,42 @@
+Updating an App's credentials
+=============================
+
+Overview
+--------
+
+You may need to change an App's credentials due to internal rotation policies or otherwise. The StreamAlert CLI allows you to easily update App credentials.
+to aid in this process, the CLI also give you the ability to list currently configured Apps.
```

Comment:
weird newline and the sentence seems to have been chopped?

Reply:
@mime-frame rst treats new lines as non-existent if the text continues on the line following. This makes the docs easier to read/edit in a code editor instead of a paragraph being on a single very long line and does not affect how the output looks